NutriLife/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+import { MantineProvider } from "@mantine/core";
+const searchBar = () =>{
+
+const[searchInput, setSearchInput] = useState("null")
+
+const handleChange = (e) => {
+    e.preventDefault();
+    setSearchInput(e.target.value);
+};
+
+return(
+    <MantineProvider>
```

Comment:
The MantineProvider tag is supposed to be the root tag for Home.jsx file and not other components. Other components should have just either a parent div tag or fragment.